src/CurveInput/index.js

```diff
@@ -501,33 +500,33 @@ class CurveInput extends Container {
             this._renderPicker();
         });
 
-        var resetZoomTooltip = new Tooltip({
-            title: 'Reset Zoom'
-        });
-        resetZoomTooltip.attach({
-            target: this._resetZoomButton
-        });
+        // var resetZoomTooltip = new Tooltip({
+        //     title: 'Reset Zoom'
+        // });
+        // resetZoomTooltip.attach({
+        //     target: this._resetZoomButton
+        // });
```

Comment:
Any reason not to delete all this commented out code?

Reply:
As this is in an unused component, perhaps we should remove the entire file for now until it's ready to be worked on?